src/util.rs

```diff
@@ -262,6 +262,15 @@ fn pop_base_face<'a>(stack: &'a mut Vec<String>, default: &'a str) -> &'a str {
     get_base_face(stack, default)
 }
 
+const FACE_DEFAULT: &str = "InfoDefault";
+const FACE_HEADER: &str = "InfoHeader";
+const FACE_BLOCK: &str = "InfoBlock";
+const FACE_LIST_ITEM: &str = "InfoBullet";
+const FACE_LINK: &str = "InfoLink";
+const FACE_MONO: &str = "InfoMono";
+const FACE_LINK_MONO: &str = "InfoLinkMono";
+const FACE_RULE: &str = "InfoRule";
```

Comment:
I'm not quite sure about the names yet.

Keeping it generic allows reuse with other plugins that might want to render markup in the info box, and reduces the number of faces a user/scheme creator needs to maintain.
But it also limits customizability and prevents giving kak-lsp's output different colours than those other plugins.